client/src/components/replyEntry.jsx

```diff
@@ -0,0 +1,70 @@
+import React from 'react';
+
+// const ReplyEntry = ({ reply }) => (
+//   <div>
+//     <div className="CM-replyEntry">
+//       <img className="CM-profileImage2" src={reply.name[0].image} alt="" />
+//       <div className="CM-userName2">{reply.name[0].name}</div>
+//       <div className="CM-commentText2">{reply.text}</div>
+//       <div className="CM-time2">{reply.time}</div>
+//     </div>
+//   </div>
+// );
+
+class ReplyEntry extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      time: null,
+    };
+    this.getTime = this.getTime.bind(this);
+  }
+
+  componentDidMount() {
+    this.interval = setInterval(() => this.getTime(), 1000);
+  }
+
+  getTime() {
```

Comment:
replyEntry and commentsEntry (btw you should rename the files so they're capitalized) each have its own getTime method that are fairly expensive... consider moving the method up to App instead and pass the method down to each class to reduce the redundancy :)

Reply:
okay